### PR TITLE
Fastly on Pantheon Guide

### LINKED ISSUE #6402: New Doc: Splunk Integration
## Summary

Add doc for Splunk integration. 

Priority: Low

## Issue Description

* Add overview of Splunk integration, following notes on https://pantheon.io/integrations/logging. "Pantheon provides access to all website application containers via SFTP. An external forwarding server can be setup to periodically collect this data from all application containers and send to Splunk. "
* Splunk log aggregation using Terminus to gather sftp connection details in a loop through all sites. (@rachelwhitton)
* Splunk integration with New Relic.
* Automate downloading of logs, then import to Splunk manually.

### Context

* From CSE Support discussion

## Suggested Resolution

* 

## Additional Information

*

### PR DESCRIPTION
**Pull request by whitneymeredith**
Closes #6402 

## Summary

**[Fastly on Pantheon Guide](https://pantheon.io/docs/guides/fastly-pantheon)** - A new guide with information on how Fastly can be used on the Pantheon platform, including integration with Splunk and New Relic.

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7390](https://pr-7390--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `41` | `45` | `4` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `98` | `0` |

Lighthouse Audit Reports:

- [`pr-7390` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/37966/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/37966/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7390--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by whitneymeredith** [comment]
[Preview](https://pr-7390--pantheon-docs.my.pantheonfrontend.website/docs/guides/fastly-pantheon)

File: source/content/guides/fastly-pantheon/01-introduction.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+anchorid: fastly-pantheon
+---
+
+Pantheon partners with [Fastly](https://www.fastly.com/) to bring you a fast and safe content delivery network (CDN).
+
+Using Fastly on the Pantheon platform to manage your site provides many benefits, including:
+
+- Building logic at the edge
+- Content personalization through [Edge Integrations](/guides/edge-integrations/) 
+- Integration with [Splunk](https://www.splunk.com/) and Splunk integration with [New Relic](/new-relic)
```
Comment (joebortelpantheon) [inline source/content/guides/fastly-pantheon/01-introduction.md:20]: Is there a particular reason we're only referencing Splunk and New Relic?

File: source/content/guides/fastly-pantheon/01-introduction.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- Enhanced Caching at the edge, including API responses. This allows you to see traffic and deployment changes, and improves continuous integration / continuous delivery (CI/CD).
+- Image Optimizer (IO) as an add-on that renders images using high-density points of presence (POPs) that cache content for longer times and serve millions of image variations. Serving from the edge in this way offloads work from your servers and improves your site's performance. IO supports several formats including animated GIFs.
+
+## Take Advantage of Fastly with Pantheon's Advanced Global CDN 
+
+Your website must be connected to Pantheon’s [Advanced Global CDN](/guides/professional-services/advanced-global-cdn) to use Fastly on the Pantheon platform. You should consult the [Edge Integrations Guide](/guides/edge-integrations/) and complete the appropriate configuration for your WordPress or Drupal site if you plan on using content personalization in addition to Fastly.
```
[the remark this answers is not in the document]
Comment (whitneymeredith) [reply source/content/guides/fastly-pantheon/01-introduction.md:26]: This guide is focused on what our customers can do with Fastly.

File: source/content/guides/fastly-pantheon/01-introduction.md (hunk 1 above)
Comment (whitneymeredith) [reply source/content/guides/fastly-pantheon/01-introduction.md:20]: @joebortelpantheon The original issue created for this only referenced Splunk and New Relic. Are there other integrations that our customers use frequently with their Fastly and Pantheon accounts that we can add to this guide?

**Review by michellecolon-pantheon — APPROVED** [review]

**Pull request merged**